README.md

```diff
@@ -1 +1,87 @@
+
+Configure Cloud Access Security Broker(CASB) connector such as Symantec CloudSOC to capture events of policy violation like 'External File Sharing'. It creates an alert of type 'Data Leakage', and then triggers a response workflow.
+
+Refer to Simulation Scenarios - Data Leakage to experience the use case without CASB configuration
+
+This article describes the “Data Leakage Threat Response” Solutions Pack. The solution pack demonstrates the scenarios and use cases around Data Leakage.
+The below scenario demonstrates and generates a demo alert for the Alert.
+
+- An alert is generated once detected a user shares a file externally, which is a violation of policy
+- This triggers an investigation that determines if the employee is on the watchlist and if the file being shared is a sensitive document
+- In case a file is found to be a sensitive document then deactivate the user in AD and revoke file permission
+- Search the incident in DLP and fetch the incident details. Create additional alerts if needed
+
+### Usage
+
+This Solution Pack ships with the following simulation scenarios. [Refer](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to Simulate Scenario documentation to understand how to Simulate and Reset Scenario.
+
+#### 1. Scenario: Data Leakage
+
+The scenario generates a demo alert of Type 'Data Leakage'.
+
+Goto generated alert and observe the following:
+
+- Reported information (policy violation action/description, violators username) is presented for analyzing the case
+- Recommendation from CloudSOC
+
+**Investigate Data Leakage Alert (Symantec CloudSOC)**: Launch the "Investigate Data Leakage Alert (Symantec CloudSOC)" playbook and observe various investigation activities such as
+
+- Identify employee watchlist status of user involved in data leakage
+- Identify if the file involved is a sensitive document
+- Initiate containment actions for sensitive document
+- Block the user in AD and investigate if the user involved in any other DLP event
+
+## Prerequisite
+
+Ensure that the below solution packs are deployed:
+
+**Solution Pack**|**Purpose**|**Doc Link**|
+| :- | :- | :- |
+|SOAR Framework 1.0.0|Require for Incident Response modules and Action playbooks|[Click here](https://github.com/fortinet-fortisoar/solution-pack-soar-framework/blob/develop/README.md)|
+|SOC Simulator 1.0.1|Require for Scenario Module and SOC Simulator connector| [Click here](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/README.md)|
+
+## Contents
+
+1. Connector(s)
+
+    |**Sr.No.**|**Connector**|
+    | :- | :- |
+    |1|Active Directory|
+    |2|Symantec CloudSOC|
+    |3|Symantec DLP|
+
+    **Warning:** After deployment, this Solution Pack will install or upgrade the stated connectors list.
+
+2. Modules Schema(s)
+    - Employee Watchlist
+    - Sensitive Files
+
+3. Role(s)
+    - Full App Permissions
```

Comment:
Why this SP needs a full app permission?

Reply:
In this solution pack, the below modules are added. Required Full app permission role for the same

    - Employee Watchlist
    - Sensitive Files